department-of-veterans-affairs/caseflow
# Make Regional Office field editable

Resolves #15060 

### Description
Updates logic to display the regional office dropdown for Virtual as well as regional office

### Acceptance Criteria

### Testing Plan
1. Shadow user `BVASYELLOW`
1. Navigate to the schedule veterans page: http://localhost:3000/hearings/schedule/assign
1. Select a regional office from the dropdown, then select the tab for AMA hearings
1. Select one of the appeals to navigate to the Case Details page for that appeal
1. Select the `Schedule Veteran` action from the dropdown menu
1. Change the type to `Virtual` and make sure you can now change the regional office dropdown
1. Submit the form and then go back to the schedule veterans queue for the RO you just scheduled for to check if they are now in the `Scheduled Veterans` tab

### User Facing Changes

 BEFORE|AFTER
 ---|---
![BeforeRODropdown](https://user-images.githubusercontent.com/61989526/95242510-20c99b80-07c4-11eb-8f23-3f816ff2a28c.png)|![AfterRODropdown](https://user-images.githubusercontent.com/61989526/95242542-2c1cc700-07c4-11eb-9eca-e14c94be8e56.png)
